docs/guide/using-validator.md

```diff
@@ -165,7 +165,7 @@ $result = (new Validator())->validate($value, $rule);
 
 ### Providing rules via dedicated object
 
-Could be helpful for reusing the same set of rules across different places. Two ways are possible - using PHP attributes 
+Could help reuse the same set of rules across different places. Two ways are possible - using PHP attributes 
```

Comment:
What's wrong with "be helpful for reusing"?

Reply:
It is longer and more complicated.